cli/src/cli_util.rs

```diff
@@ -545,7 +545,7 @@ impl CommandHelper {
                 // merged repo wouldn't change because the old one wins, but it's probably
                 // fine if we picked the new wc_commit_id.
                 let stats = workspace_command
-                    .maybe_snapshot_impl(ui)
+                    .maybe_snapshot_impl(ui, /* skip_head_reload: */ true)
```

Comment:
Perhaps, we shouldn't do `import_git_head()` here because we intentionally snapshot the working copy at old operation.